examples/features/metadynamics/pimd_metadynamics_zundel/input.xml

```diff
@@ -33,7 +35,9 @@
       <ensemble>
          <temperature units="kelvin"> 300.0 </temperature>
          <bias>
-            <force forcefield="plumed" nbeads="1"></force>
+           <force forcefield="plumed" nbeads="1">
+             <force_extras> [ doo, mtd.bias ] </force_extras>
```

Comment:
So this uses ring polymer "expansion" allowing beadwise printout of extras? What if I am running PIMD and I want to print out the extras for the centroid?

Reply:
Well, you have to compute it, so you would add a "silent" force component
```
<force forcefield="plumed" nbeads="1" weight="0.0"></force>
```
and reference it from the `<outputs>` section with `potential_component(idx)` properties and trajs. That way it'll be computed only when it's required to print it out.

Reply:
I guess this is not entirely clear to me (an experienced user), so I think it will also be unclear for other practitioners. My confusion is twofold:

1. How one can access the extras from a given 'force component'. Do we have an example for this? 

2. Isn't the extras like any trajectory-compatible quantity represented internally beadwise? So even if `<force forcefield="plumed" nbeads="1" weight="0.0"></force>` calculates the forcefield (forces and extras), wouldn't the print option only be available for beads? I don't see, other than the case for classical MD where beads and centroid are the same, how one can print out extras for a centroid.

Maybe this use case needs an example.

Reply:
yes, there are examples in `examples/temp/pes/pswater/combine-extras`. still need to be ported somewhere inside the "canonical" examples - feel free to help ^_^